resource/command.go

```diff
@@ -56,18 +57,53 @@ func (c *Command) Validate(sys *system.System) []TestResult {
 
 	var results []TestResult
 	sysCommand := sys.NewCommand(c.GetExec(), sys, util.Config{Timeout: time.Duration(c.Timeout) * time.Millisecond})
+	newSysCommand := &NewSysCommand{sysCommand, []byte{}, []byte{}}
+	newSysCommand.ReadStreams()
```

Comment:
In original version, `sysCommand.Stdout` and `sysCommand.Stderr` methods may not be called at, or only one of them called - if passed as parameter to `ValidateContains` call below. With this change, we wrap `sysCommand` into another `newSysCommand` structure, which always reads bytes from `Stdin` and `Stdout` methods, and keeps them. If `ValidateContains` needs content of stdout/stderr, it gets `GetStdoutBytesReader`/`GetStderrBytesReader` method which also returns `io.Reader` object as requested.